MapboxNavigation/StatusView.swift

```diff
@@ -46,6 +46,26 @@ public class StatusView: UIControl {
         }
     }
     
+    var statuses: [Status] = []
+
+    public struct Status: Identifiable {
+        public var id: String
+        var spinner: Bool = false
+        let duration: TimeInterval
+        var animated: Bool = true
+        var interactive: Bool = false
+        var priority: Priority
+    }
+    
+    struct Priority: RawRepresentable {
```

Comment:
It'd be better to use enum for status priority.

Reply:
I think the idea was to allow for arbitrary priorities, similar to constraint priorities. But making the type public would make sense.